docs/content/downloads/releasenotes/release-0.7.0.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "0.7.0"
+menu:
+  main:
+    parent: "ReleaseNotes"
+    weight: 70
+---
+
+* [Release log](https://projects.eclipse.org/projects/modeling.elk/releases/0.7.0)
+* [Documentation](https://download.eclipse.org/elk/updates/releases/0.6.1/elk-0.7.0-docs.zip)
```

Comment:
I guess this should be https://download.eclipse.org/elk/updates/releases/0.7.0/elk-0.7.0-docs.zip